setup.py

```diff
@@ -59,5 +62,4 @@ def readme():
     url=URL,
     packages=find_packages(),
     use_scm_version=USE_SCM_VERSION,
-    setup_requires=SETUP_REQUIRES,
```

Comment:
This is handled in the pyproject.toml now. We can move pretty much everything from the setup.py to the toml and cfg files.